<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xToolsSerialPortTransmitterToolUiEditor</class>
 <widget class="QWidget" name="xToolsSerialPortTransmitterToolUiEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Port Transmitter Item Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Parity</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Stop bits</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="xToolsFlowControlComboBox" name="comboBoxFlowControl"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxEnable">
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="xToolsBaudRateComboBox" name="comboBoxBaudrate">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="xToolsParityComboBox" name="comboBoxParity"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Baudrate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Data bits</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="xToolsStopBitsComboBox" name="comboBoxStopBits"/>
   </item>
   <item row="1" column="1">
    <widget class="xToolsPortNameComboBox" name="comboBoxPortName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Port name</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Flow control</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="xToolsDataBitsComboBox" name="comboBoxDataBits"/>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xToolsPortNameComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsPortNameComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsBaudRateComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsBaudRateComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsDataBitsComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsDataBitsComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsStopBitsComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsStopBitsComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsParityComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsParityComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsFlowControlComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsFlowControlComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
